scripts/globals/bcnm.lua

```diff
@@ -921,6 +927,12 @@ local function checkReqs(player, npc, bfid, registrant)
             return player:hasKeyItem(xi.ki.ZEPHYR_FAN) and npcId == getEntranceOffset(4)
         end,
 
+        [ 643] = function() -- ENM: Brothers
```

Comment:
you'll also need to put these lines around line ~1250

Reply:
?